portfolio/src/main/webapp/script.js

```diff
@@ -103,3 +106,35 @@ function deleteComment(comment) {
     getComments()
   );
 }
+
+function findSkill(num) {
+  let skills = [
+    "skills/cpp-skill.html",
+    "skills/py-skill.html",
+    "skills/java-skill.html",
+    "skills/html-skill.html",
+    "skills/css-skill.html",
+    "skills/js-skill.html",
+    "skills/git-skill.html",
+    "skills/bash-skill.html",
+    "skills/matlab-skill.html",
+    "skills/gcp-skill.html",
+    "skills/sql-skill.html",
+    "skills/office-skill.html",
```

Comment:
I feel like you could use a stringbuilder to build the path since they're all "skills/< >-skill.html"

Reply:
done